privacy-snapshot.json

```diff
@@ -52,12 +52,13 @@
   "security-alerts.api.cx.metamask.io",
   "security-alerts.dev-api.cx.metamask.io",
   "sentry.io",
+  "sepolia.infura.io",
```

Comment:
we'd prefer to mock all places hitting this.  I think we expected https://github.com/MetaMask/metamask-extension/pull/28277 to cover all places and, aren't sure which caller remains.

Reply:
Agree; Inspecting the network tab on the test `test/e2e/tests/settings/account-token-list.spec.js`; the calls to infura that i see are `eth_blockNumber` even though i added a mock for those, it is still requiring the snapshot update; not quite sure what i missed there